packages/venia-ui/lib/components/CheckoutPage/PaymentInformation/paymentInformation.js

```diff
@@ -60,17 +61,20 @@ const PaymentInformation = props => {
     const paymentInformation = doneEditing ? (
         <Summary onEdit={showEditModal} />
     ) : (
-        <PaymentMethods
-            onPaymentError={handlePaymentError}
-            onPaymentSuccess={handlePaymentSuccess}
-            resetShouldSubmit={resetShouldSubmit}
-            shouldSubmit={shouldSubmit}
-        />
+        <Form>
+            <PaymentMethods
+                onPaymentError={handlePaymentError}
+                onPaymentSuccess={handlePaymentSuccess}
+                resetShouldSubmit={resetShouldSubmit}
+                shouldSubmit={shouldSubmit}
+            />
+        </Form>
```

Comment:
Why did we move the form down a level from the wrapper? Is it because `Dialog` has its own internal form? Was there information shared between the `PaymentMethods` and the `EditModal` through form state/api that we need to make sure is still shared?

Reply:
If I move `resetShouldSubmit` & `shouldSubmit` to `editModal` props, it can not click to edit button because the credit card form alway run